rules/opentitan/fpga_cw310.bzl

```diff
@@ -226,6 +228,8 @@ def cw310_params(
       rom_ext: Use an alternate ROM_EXT for this test.
       otp: Use an alternate OTP configuration for this test.
       bitstream: Use an alternate bitstream for this test.
+      clear_bitstream: Whether to clear the bitstream before running the test.
+      need_jtag: If this test requires JTAG access, set this to "cw310" or "hyper310".
```

Comment:
I wonder if this should be a boolean and it's up to the exec_env to decide how to provide the jtag? Maybe this can be done as part of future refactoring.